src/store/Reducer.js

```diff
@@ -109,6 +109,41 @@ const Reducer = (state, action, value = action && action.data && action.data.val
       };
     }
 
+    case 'UPDATE_STATE': {
+      if (action.data.council === 'NECCC') {
+        return {
+          ...state, state: action.data.state, council: 'Northeast', stateId: action.data.stateId,
```

Comment:
seems like maybe the long name "Western" vs "wccc" should also be returned by the api so this doesn't need to be hard coded here